backend/flakestry/api/publish.py

```diff
@@ -15,7 +15,8 @@
 
 
 class Publish(BaseModel):
-    version: str
+    ref: str | None
```

Comment:
we'll need a migration for this, I need to look into #7

Reply:
Why? Isn't this about the JSON payload for the API endpoint? I don't see how it would break the publish github action. And for the db it uses `Release` instead.